pkg/parser/parser.go

```diff
@@ -3,8 +3,11 @@ package parser
 import (
 	"bytes"
 	_ "embed"
+	"encoding/json"
```

Comment:
can you try the Golang JSON serializer with an enum field? I think it serializes the enum to its numeric value instead, might be worth checking if it deserializes it back correctly though. SSHStatus might be a good one

Reply:
added DISABLE_SSH. PTAL